README.md

```diff
@@ -1,2 +1,65 @@
-# portofolio_site
-this is my portofolio site
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portofolio Site
+
+> this is the mobile version of my portofolio site 
+
+
+## Built With
+- Major languages
+- Frameworks
+- Technologies used
+
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+**This is my Day 2 at Microverse project**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+    Web Browser (e.g. Firefox, Chrome, Edge)
+
+### Setup
+    Download the code from this repository.
+
```

Comment:
- [ ] Kindly state the steps required to get the project to a local machine (In your own language, state the steps to download the repo and get it up and running in the local machine 😄 ). Your readme will look very professional this way and beneficial for anyone who wants to use or work on your repo👍🏼